### DISCUSSION TITLE
Committer Identity Unkown

### DISCUSSION THREAD

**Discussion by spollacs** [requarks/wiki#7062] [Error / Bug Report] [answered]
hi, i am currently getting this error "Committer Identity Unknown, please tell me who you are..." whenever I try to save an edited file. 

![committer](https://github.com/requarks/wiki/assets/54198789/47ef425b-cc8f-4050-a62d-1bb21a7c7248)

Whenever I click save, it shows that error but when I exit and discard changes, the file still got saved despite of the error given.

How can I remove this?

**Comment by NGPixel** [comment, marked as answer]
You need to set the Default Author Email and Name in your git storage config.
